ee/api/test/test_billing.py

```diff
@@ -1071,7 +1071,7 @@ def test_activate_success(self):
         self.assertRegex(response.url, url_pattern)
 
     def test_deprecated_activation_success(self):
```

Comment:
I think this whole test needs to be removed?

Reply:
No I'm keeping the /activation endpoint live for now. I will go back and remove it after I've confirmed there are no requests being made to it.

Reply:
This test is there to make sure the deprecated URL is still working.